src/DocumentFormat.OpenXml/Properties/Properties.cs

```diff
@@ -18,6 +18,9 @@
 [assembly: InternalsVisibleTo("DocumentFormat.OpenXml.CodeGeneration.Schemas, Publickey=002400000480000094000000060200000024000052534131000400000100010061d8931836c82bf25ca6b773dfd6e7b3ab4e43fba60cf4a86347170373415a165ccc40da3da4a52163822db9fa91f15828236d32d6a9fe754859f10d1f8262646c1f3fb6b4348123f14d733db0ff11c3198b7cf56caaebbf14563990446a6c32aff36d5a7097194294c127fe3cdf9f2609daae5f4daf26f8b6227f203d2a8bbf")]
 [assembly: InternalsVisibleTo("DocumentFormat.OpenXml.CodeGeneration.Schematron, Publickey=002400000480000094000000060200000024000052534131000400000100010061d8931836c82bf25ca6b773dfd6e7b3ab4e43fba60cf4a86347170373415a165ccc40da3da4a52163822db9fa91f15828236d32d6a9fe754859f10d1f8262646c1f3fb6b4348123f14d733db0ff11c3198b7cf56caaebbf14563990446a6c32aff36d5a7097194294c127fe3cdf9f2609daae5f4daf26f8b6227f203d2a8bbf")]
 
+// For Linq namespace class genertion
+[assembly: InternalsVisibleTo("DocumentFormat.OpenXml.CodeGeneration.Linq, Publickey=002400000480000094000000060200000024000052534131000400000100010061d8931836c82bf25ca6b773dfd6e7b3ab4e43fba60cf4a86347170373415a165ccc40da3da4a52163822db9fa91f15828236d32d6a9fe754859f10d1f8262646c1f3fb6b4348123f14d733db0ff11c3198b7cf56caaebbf14563990446a6c32aff36d5a7097194294c127fe3cdf9f2609daae5f4daf26f8b6227f203d2a8bbf")]
```

Comment:
I would love to see some of the internal types start to be made public, so we can stop needing so many internalsvisibleto. If the shape of the APIs being used are working well, we can look at removing that for this project at least.

Reply:
Agree. I only had to add this because the relevant types are internal at the moment.